packages/react/src/components/Backdrop/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Let's remove the phrase All Rights Reserved in the new files as well.

```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). 
```